### PR TITLE
Favicon

### PR DESCRIPTION
**Pull request by BrandonKlotz**

### CODE REVIEW

File: 501c3.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
     <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
+    <link rel="shortcut icon" href="https://raw.githubusercontent.com/citizenlabsgr/homepage/blob/master/brand-assets/Logo/favicon.png" />
```
Comment (jacebrowning) [inline 501c3.html]: I think you want this link to be: https://raw.githubusercontent.com/citizenlabsgr/homepage/master/brand-assets/Logo/favicon.png

Comment (jacebrowning) [inline 501c3.html]: See https://raw.githubusercontent.com/citizenlabsgr/homepage/master/images/logo.png as an example.

Comment (jacebrowning) [inline 501c3.html]: Or try `href="/brand-assets/Logo/favicon.png"`.

**Comment by jacebrowning** [comment]
It's working!

Now it just needs to be a little bit bigger...



I've head good luck with http://realfavicongenerator.net/.

**Comment by BrandonKlotz** [comment]
@jacebrowning I am not sure why it is still appearing so small I made the file size 48px x 48px

**Comment by BrandonKlotz** [comment]
It now looks much better! @jacebrowning

**Comment by jacebrowning** [comment]
This is what it looks like for me:

screen shot 2017-05-03 at 11 36 31 am

It's still quite a bit smaller than other favicons. What do you think?

**Comment by BrandonKlotz** [comment]
screen shot 2017-05-03 at 11 38 00 am

@jacebrowning this is how mine appears.

**Comment by Allen616** [comment]
I am not seeing a favicon.

**Comment by BrandonKlotz** [comment]
@Allen616 What browser are you viewing it in? Could you snap a screenshot?

**Comment by jacebrowning** [comment]
Here's a report of multiple browsers: http://realfavicongenerator.net/favicon_checker?protocol=https&site=citizenlabsgr-homepage-pr-21.herokuapp.com%2F

**Comment by BrandonKlotz** [comment]
Just found that! I am going to update the code and see if this helps! Good find!

**Comment by BrandonKlotz** [comment]
screen shot 2017-05-03 at 11 53 12 am
screen shot 2017-05-03 at 11 53 20 am

@Allen616 @jacebrowning Can you review from your side to see if this last commit has worked?

**Comment by Allen616** [comment]
Works for me...

favicon

**Pull request merged**